pkg/apis/core/types.go

```diff
@@ -305,21 +305,34 @@ type PersistentVolume struct {
 
 // PersistentVolumeSpec has most of the details required to define a persistent volume
 type PersistentVolumeSpec struct {
-	// Resources represents the actual resources of the volume
+	// A description of the persistent volume's resources and capacity.
```

Comment:
I'm looking at the other capacity fields, it looks like the descriptions there are more along the lines of "Represents the available capacity of a ..."